app/view_models/winners_view_model.rb

```diff
@@ -8,38 +8,40 @@ def upcoming_count
   end
 
   def total_auctions
-    auction_count
+    published_auction_count
   end
 
   def unique_auction_winners
-    completed_auctions.map do |auction|
+    successful_auctions.map do |auction|
       WinningBid.new(auction).find.bidder
     end.uniq.count
   end
 
   def average_bids_per_auction
-    if auction_count > 0
-      completed_auctions.map(&:bids).flatten.count / auction_count
+    if successful_auction_count > 0
+      successful_auctions.map(&:bids).flatten.count / successful_auction_count
     else
       'n/a'
     end
   end
 
   def vendors_with_bids_count
-    User.includes(:bids).where(bids: { bidder_id: nil }).count
+    User.includes(:bids).where.not(bids: { bidder_id: nil }).count
```

Comment:
Hrmm ... what about getting all bids, map to vendor, then uniq? Just thinking out loud.